test/dlopen/test_dlopen.c

```diff
@@ -17,6 +17,8 @@ Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
 #include <stdio.h>
 #include "gotcha/gotcha.h"
 
+# define RTLD_NEXT	((void *) -1l)
+# define RTLD_DEFAULT	((void *) 0)
```

Comment:
We shouldn't have to redefine these. They should come from including dlfcn.h

Reply:
i forgot to commit the final. I meant to define ```_GNU_SOURCE``` instead.